src/styles.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
```

Comment:
What is the benefit of using `styled-components`? It seems like everything you're using it for right now could be done with simple CSS and would result in a clearer `Dropdown.js`.

Reply:
Will remove this file and import the styling from our CSS style file